CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+## 1.7.0-beta.1
+
+## ✨ Features
+* Add a new constructor option to `mapboxgl.Popup`, `closeOnMove`, that closes the popup when the map's position changes ([#9163](https://github.com/mapbox/mapbox-gl-js/pull/9163))
+* Add third parameter to once event handler ([#8875](https://github.com/mapbox/mapbox-gl-js/pull/8875))
```

Comment:
```
`map.once()` now allows specifying a layer-idm as a third parameter making it consistent with `map.on()`.
```

Reply:
Added as 'layer id' instead of layer-idm